core/build.gradle

```diff
@@ -28,7 +28,12 @@ ext {
 }
 
 tasks.register('versionWriterTask') {
-    def versionName = project.property("VERSION_NAME")
+    String versionName
+    if (project.hasProperty("VERSION_NAME")) {
+        versionName = "${project.property("VERSION_NAME")}"
+    } else {
+        versionName = project.version
+    }
```

Comment:
This project will use `VERSION_NAME` during publish, dbapp has no visibility on `gradle.properties` as it has its own. We use `project.version` as a fallback and set it within dbapp.